pyqtgraph/metaarray/MetaArray.py

```diff
@@ -663,7 +663,6 @@ def prettyInfo(self):
                     colstrs.append(cs)
                 axs += '[' + ', '.join(colstrs) + ']'
             s += axs + "\n"
-        s += str(self._info[-1])
```

Comment:
..and this should be:

    if len(self._info) > self.ndim:
        s += str(self._info[-1])